scripts/upgrade

```diff
@@ -77,6 +77,9 @@ ynh_install_php7
 
 # Download, check integrity, uncompress and patch the source from app.src and do required installation for update
 
+# version before upgrade
+old_ver=`sudo yunohost app info $app --output-as plain | ynh_get_plain_key version`
```

Comment:
I took different approach for taking out version number from the core app file. But your way is much better and appropriate to yunohost way.
So I request you to remove the code on line 84 and keep your code at 81 as default way to get version number.